tests/brownie/token/.github/workflows/main.yaml

```diff
@@ -0,0 +1,42 @@
+on:
+  push:
+  pull_request:
+  schedule:
+    # run this workflow every Monday at 1PM UTC
+    - cron: "* 13 * * 1"
+
+name: main workflow
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Cache Solidity Installations
+      uses: actions/cache@v1
+      with:
+        path: ~/.solcx
+        key: ${{ runner.os }}-solcx-cache
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+
+    - name: Install Ganache
+      run: npm install -g ganache-cli@6.8.2
```

Comment:
why do you need ganache-cli? It's always bin only a dependency of other dependencies

Reply:
@apetrovskiy i remove this, it's not our github workflow (didn't see this in new files)